ops/op2/dispatch_slow.rs

```diff
@@ -360,6 +365,47 @@ pub fn from_arg(
   Ok(res)
 }
 
+pub fn from_arg_buffer(
+  generator_state: &mut GeneratorState,
+  arg_ident: &Ident,
+  buffer: &Buffer,
+) -> Result<TokenStream, V8MappingError> {
+  let throw_exception =
+    throw_type_error(generator_state, "expected buffer".to_string())?;
```

Comment:
Isn't this eagerly evaluating the type error?

Reply:
Nope -- this is at generation time. It eagerly evaluates the string, but I'm not worrying too much about allocations here (a few hundred ops should process in milliseconds even if we're lazy about it).